xml/System.Formats.Tar/PaxTarEntry.xml

```diff
@@ -67,37 +66,30 @@ The entry type of <paramref name="other" /> is not supported in the PAX format.<
         <summary>Initializes a new <see cref="T:System.Formats.Tar.PaxTarEntry" /> instance with the specified entry type and entry name, and the default extended attributes.</summary>
         <remarks>
+          <para>When creating an instance using the <see cref="M:System.Formats.Tar.PaxTarEntry.#ctor(System.Formats.Tar.TarEntryType,System.String)"/> constructor, only the following entry types are supported: <see cref="F:System.Formats.Tar.TarEntryType.Directory"/>, <see cref="F:System.Formats.Tar.TarEntryType.HardLink"/>, <see cref="F:System.Formats.Tar.TarEntryType.SymbolicLink"/>, <see cref="F:System.Formats.Tar.TarEntryType.RegularFile"/>, <see cref="F:System.Formats.Tar.TarEntryType.BlockDevice"/>, <see cref="F:System.Formats.Tar.TarEntryType.CharacterDevice"/>, and <see cref="F:System.Formats.Tar.TarEntryType.Fifo"/>.</para>
+          <para>Use the <see cref="M:System.Formats.Tar.PaxTarEntry.#ctor(System.Formats.Tar.TarEntryType,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.String}})" /> constructor to include additional extended attributes when creating the entry.</para>
+          <para>The following entries are always found in the extended attributes dictionary of any PAX entry:</para>
           <format type="text/markdown"><![CDATA[
-
-## Remarks
-
-When creating an instance using the <xref:System.Formats.Tar.PaxTarEntry.%23ctor(System.Formats.Tar.TarEntryType,System.String)> constructor, only the following entry types are supported:
-
-
-- In all platforms: <xref:System.Formats.Tar.TarEntryType.Directory>, <xref:System.Formats.Tar.TarEntryType.HardLink>, <xref:System.Formats.Tar.TarEntryType.SymbolicLink>, <xref:System.Formats.Tar.TarEntryType.RegularFile>.
-- In Unix platforms only: <xref:System.Formats.Tar.TarEntryType.BlockDevice>, <xref:System.Formats.Tar.TarEntryType.CharacterDevice> and <xref:System.Formats.Tar.TarEntryType.Fifo>.
-
-Use the <xref:System.Formats.Tar.PaxTarEntry.%23ctor(System.Formats.Tar.TarEntryType,System.String,System.Collections.Generic.IEnumerable{System.Collections.Generic.KeyValuePair{System.String,System.String}})> constructor to include additional extended attributes when creating the entry.
-
-The following entries are always found in the Extended Attributes dictionary of any PAX entry:
-
 - Modification time, under the name `mtime`, as a <xref:System.Double> number.
 - Access time, under the name `atime`, as a <xref:System.Double> number.
 - Change time, under the name `ctime`, as a <xref:System.Double> number.
 - Path, under the name `path`, as a string.
-
-The following entries are only found in the Extended Attributes dictionary of a PAX entry if certain conditions are met:
-
+          ]]></format>
+          <para>The following entries are only found in the extended attributes dictionary of a PAX entry if certain conditions are met:</para>
+          <format type="text/markdown"><![CDATA[
```

Comment:
I would just combine this with the previous CDATA section.

Reply:
Also I think the warnings are indicating that you can't combine XML and HTML in the remarks, so just pick one.

Reply:
Merged.